v2/devices/axolotl.yml

```
@@ -25,6 +25,18 @@ user_actions:
     description: "With the device powered off, hold Volume Down + Power."
     image: "phone_power_down"
     button: true
+  shiftos_reboot_system:
+    title: "Recovery - Reboot system now"
+    description: "With the recovery mode active, navigate using your volume keys up/down and select 'Reboot system now'.\nConfirm with pressing the Power key.\n\nWarning: Do NOT lock your bootloader, this is not supported yet."
```

`\n` does not work.

Is this something we want to support?

---

Dunno. Do __you__ want us to support it? :D

---

Would be cool to split up large blocks of text. :)

But i don't know how much work is involved to realize that and if it fits the UX vision of the project.

Personally I think the more we encourage custom text and make it more flexible to use, the harder it will be to add translations in future.

---

guess we can add markdown support there. but we should make sure not to let it get out of hand. for now, try using `<br>` tags instead of `\n`, that should work.
```suggestion
    description: "With the recovery mode active, navigate using your volume keys up/down and select 'Reboot system now'.<br>Confirm with pressing the Power key.<br><br>Warning: Do NOT lock your bootloader, this is not supported yet."
```